### ISSUE TITLE
Add a forum to support the Raku community and provide a searchable repository of Q&A

### ISSUE THREAD

**Issue by shishini** [Raku/problem-solving#251]
### Suggestion

This is a request to add and fund a discourse forum to serve the Raku community
I will explain below why adding one is a good idea, if you think you like the idea
and don't to listen to me rambling why we need one, please go directly to cost and 
implementation options

### Justification

We don't have an easy to search repository of question and answers

Currently the Raku Community page (https://www.raku.org/community/)
seem to only really offer one option to interested developers to
communicate with the community, the IRC channel on freenode 

The discord server seem to be just a bridge to the IRC channel
and it remains a chatting option 

The mailing list is said to be inactive as of 2014 

New comers, don't have access to any searchable repository of questions
and answer s

New comers, will have to repeat the same question on the IRC channel
or the unofficial Facebook page (https://www.facebook.com/groups/raku.perl6)

Asking in chat, can be intimidating, because of the live interaction
forums in my opinion provide a less stressful form of communication

You can ask a question, edit it, follow up on it, and you don't have to
connected live or online to wait for answer 

You can also also easily search if the same question was asked before
and use that thread instead of repeating the question

The Perl (5/7) community, have had the https://www.perlmonks.org/ ( https://perldoc.perl.org/perlcommunity#Forums )
page for years and its a very rich place to search for information about Perl 5

Other newer languages, like Go, Rust, Fsharp, Julia 
opted to use the discourse platform
It is modern, feature rich and easy to use 

You can look at it in those links 

https://forums.fsharp.org
https://discuss.ocaml.org
https://users.rust-lang.org
https://discourse.julialang.org
https://forum.golangbridge.org
https://clojureverse.org

### Cost and implementation option

We can either, install from source on our own servers
or pay for a hosted option 

A hosted option will cost 50 USD for the standard plan 
150 USD for the enterprise plan
And we will have to call if we need a bigger plan

Those costs are at 50% discount from the commercial plans
https://www.discourse.org/pricing

Considering the size of the Raku community, I think
the standard plan will be good enough

### Alternatives

- discourse is opensource, if the cost seem to much, and
if it will be cheaper to install discourse on a server 
available for the Raku community, this can be an option
I personally don't have any experience installing
or admining a discourse installation, but if needs be
I can look into it 

- User Perlmonks, historically Perlmonks was used for Perl6 
and it seem to still get some questions on Raku, although very few
We can reignite the use of Perlmonks by the Raku community 
and publish a link to it on the community page 
https://www.raku.org/community/

**Comment by JJ** [comment]
Thanks for raising this issue, and very grateful for the kind offer. The mailing list is actually rather active, with emails almost every single day. And [StackOverflow](https://stackoverflow.com/questions/tagged/raku) works rather well, with a high rate of answered questions, most of them during the day. Besides, people know that SO is *the* place for asking questions. So, in principle I'm not so totally sure we would need another place. Also, I think this is more of an infra than documentation question, so I'd rather hear @rba's opinion on this matter.

**Comment by shishini** [comment]
I agree, that if the mailing list is still active, considering the size of the community, paying for a discourse plan, is probably not very well justified 

Feel free to close this issue, a discourse forum is just nice to have, far from being a must have 
Maybe in the future if the community becomes bigger, we can reconsider 

For me personally, I will also try to post my questions on perlmonks , Raku is still  on topic on that forum

**Comment by jubilatious1** [comment]
I enjoy using the perl6-users mailing list (https://lists.perl.org/list/perl6-users.html ). 

- First of all, it's indexed and searchable within my email client.  
- Second of all, I can keep a copy of mailing list posts on my hard drive--no net connection needed.  
- Third of all, I can cite a post directly from the NNTP archive ( https://www.nntp.perl.org/group/perl.perl6.users/ ), to reference any prior topic under discussion--for example--should I be opening a Github issue. 

We hope you'll join us!

cc: @AlexDaniel

**Comment by shishini** [comment]
> We hope you'll join us!

Sure I will, the [community page](https://www.raku.org/community/) just gave me the impression that the mailing list is not active anymore, I should have checked it myself
